react/features/base/participants/middleware.js

```diff
@@ -37,30 +39,15 @@ MiddlewareRegistry.register(store => next => action => {
         break;
 
     case KICK_PARTICIPANT:
-        if (typeof APP !== 'undefined') {
-            APP.UI.emitEvent(UIEvents.USER_KICKED, action.id);
-        }
+        conference.kickParticipant(action.id);
+
         break;
 
-    case MUTE_REMOTE_PARTICIPANT:
-        if (typeof APP !== 'undefined') {
-            APP.UI.messageHandler.openTwoButtonDialog({
-                titleKey: 'dialog.muteParticipantTitle',
-                msgString:
-                    '<div data-i18n="dialog.muteParticipantBody"></div>',
-                leftButtonKey: 'dialog.muteParticipantButton',
-                dontShowAgain: {
```

Comment:
The removal of this dontShowAgain was intentional? That saves a lot of work if it's intentional. Maybe I've already said the same exact thing previously.

Reply:
Yes, it was intentional. I started advocating for not adding such checks in general.